onnx/defs/data_type_utils.h

```diff
@@ -47,6 +47,7 @@ class DataTypeUtils final {
   // If the DataType input is invalid, this function will throw std::invalid_argument exception.
   // If ONNX_NO_EXCEPTIONS is set it will abort.
   static const TypeProto& ToTypeProto(const DataType& data_type);
+  static std::string ToDataTypeString(int32_t tensor_data_type);
```

Comment:
I moved this utility function to public function to get type string from tensor_data_type for ParseData